listenbrainz_spark/recommendations/train_models.py

```diff
@@ -149,19 +146,19 @@ def main():
     time_ = defaultdict(dict)
     try:
         listenbrainz_spark.init_spark_session('Train Models')
-    except AttributeError:
-        logging.info('Aborting...')
+    except Py4JJavaError as err:
+        logging.error('{}\n{}\nAborting...'.format(str(err), err.java_exception))
```

Comment:
@mayhem wherever we see a description like this,  the detailed error and its stack trace have been raised in the called function. Here, we have room to add extra information or just display the error message. 

Places where additional info wasn't required, I have displayed the raised error (which already has a lot of info).